packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -372,6 +373,7 @@ export function toInstrumentedOdspTokenFetcher(
 	resolvedUrlParts: IOdspUrlParts,
 	tokenFetcher: TokenFetcher<OdspResourceTokenFetchOptions>,
 	throwOnNullToken: boolean,
+	returnPlainToken: boolean = false,
```

Comment:
Explain plain token here too

Reply:
I will just add jsdoc for this param, should cover both.